baby-liminal-extension/src/backend/benchmarking.rs

```diff
@@ -31,12 +47,38 @@ pub struct Pallet<T> {
 /// limits the confusion to this module only.
 pub type ChainExtensionBenchmarking<T> = Pallet<T>;
 
+/// Get the verification artifacts from the benchmark resources.
+///
+/// Since the benchmarks are run within the runtime, we don't have access to the common `std::fs` utilities.
+/// Fortunately, we can still make use of the `include_bytes` macro.
+fn get_verification_artifacts<T: pallet_vk_storage::Config>() -> (
+    VerifyArgs,
+    BoundedVec<u8, <T as pallet_vk_storage::Config>::MaximumKeyLength>,
+) {
+    // We use a macro here, because a function cannot put literal variables in the `include_bytes` macro.
```

Comment:
You could also just inline the `include_bytes!("../../benchmark-resources/5_3_proof")` and the same for `input`. 

This looks more impressive though.

Reply:
oh yes, but I wanted to avoid specifying this long path 3 times